README.md

```diff
@@ -416,20 +416,21 @@
 
 ### Modules
 
-* <a name="consistent-module-structure"></a>
-  Use a consistent structure in module definitions.
-  <sup>[[link](#consistent-module-structure)]</sup>
+* <a name="use-import-alias-require"></a>
+  Use a consistent structure when calling `use`/`import`/`alias`/`require`: call
+  them in this order and group multiple calls to each of them.
+  <sup>[[link](#use-import-alias-require)]</sup>
 
   ```elixir
-  @compile :inline_list_funcs
-
   use GenServer
 
-  require Logger
+  import Bitwise
+  import Kernel, except: [length: 1]
 
-  alias Kernel.Typespec
+  alias Mix.Utils
+  alias MapSet, as: Set
```

Comment:
Maybe some other data structure will work here (e.g. `alias RedBlackTree, as: Tree`) but to me `as: Set` looks fine.